kernel/src/grant.rs

```diff
@@ -371,11 +371,11 @@ impl<'a, T: Default, const NUM_UPCALLS: usize> ProcessGrant<'a, T, NUM_UPCALLS>
     /// relevant error.
     fn new(grant: &Grant<T, NUM_UPCALLS>, processid: ProcessId) -> Result<Self, Error> {
         // Moves non-generic code from new() into non-generic function to reduce
-        // code bloat from the generic function being monomorphized, as it is common
-        // to have over 50 copies of Grant::enter() in a Tock kernel (and thus 50+
-        // copies of this function). The returned Option indicates if the returned
-        // pointer still needs to be initialized (in the case where the grant was only
-        // just allocated).
+        // code bloat from the generic function being monomorphized, as it is
+        // common to have over 50 copies of Grant::enter() in a Tock kernel (and
+        // thus 50+ copies of this function). The returned Option indicates if
+        // the returned pointer still needs to be initialized (in the case where
+        // the grant was only just allocated).
```

Comment:
Is your editor configured to do something different that `rustfmt` somehow? There's a whole bunch of these comment line-wrapping changes [It doesn't look like any actual text changed here?] that just feel like noise in this PR.

Reply:
That was the main point of this PR :)

Reply:
rustfmt doesn't touch comments, and grant.rs has been updated enough recently with things moving in and out that comments ended up wrapped to all sorts of various widths. This does some prettyifying.

Also, it would be great if rustfmt would do this for us, but I suspect it's impossible to automatically detect what is a hardwrapped paragraph and what is intentionally formatted.

Reply:
By default, rustfmt does not enforce wrapping comments or maximum comment length.

There is an unstable feature (https://github.com/rust-lang/rustfmt/issues/3349) that can be enabled to make rustfmt enforce this, but I am not sure it is worth using unstable rustfmt just for comment formatting.

Given that rustfmt does not litigate this stuff, I think these changes are reasonable.

Reply:
Can we have `comment_min_width` too? :)